### DISCUSSION TITLE
Remove or hide "Go to profiles.ini folder" menu item.

### DISCUSSION THREAD

**Discussion by hermanbeitsma** [snappysnappydog/Simple-IP-Config#201] [Ideas]
Hi!

First of all, thank you all so much for creating and updating this great tool!
I do have a concern and request. The menu item "Go to profiles.ini folder" in the Tools menu opens the file explorer with Administrator rights which allows you to then for example start cmd.exe also with administrator rights. This is possible even if you logged in as a user without administrator rights.

This is a security risk. And therefore, I would like to ask:

-	Is it possible to remove or hide this menu item?
-	Or would it be possible to let Explorer open without the administrator rights. That would be the best option. But I do not think that is possible because it inherits the rights from the program that gives the command.

Best regards,
Herman

**Comment by JonnyTech** [comment]
There exists such a feature in Windows - `CreateRestrictedToken`

**Comment by KurtisLiggett** [comment]
@hermanbeitsma, thanks for pointing this out - I had not considered this case. I'll check into options to open explorer with restricted rights, but I may just remove it for now and add it to the list of to-do's.

**Reply by hermanbeitsma** [reply]
Hello Kurtis, thank you so much for looking into this matter!
